src/d3a/models/market/two_sided_pay_as_clear.py

```diff
@@ -27,7 +28,13 @@ class TwoSidedPayAsClear(TwoSidedPayAsBid):
     def __init__(self, time_slot=None, area=None,
                  notification_listener=None, readonly=False):
         super().__init__(time_slot, area, notification_listener, readonly)
-        self.state = MarketClearingState()
+        self.cumulative_offers = dict()  # type Dict[Datetime, dict()]
+        self.cumulative_bids = dict()  # type Dict[Datetime, dict()]
+        self.clearing = defaultdict(int)  # type: Dict[DateTime, tuple()]
+
+    @classmethod
+    def _csv_fields(cls):
+        return 'time', 'rate [ct./kWh]'
```

Comment:
I think I have left a comment in another PR too, clearing rate should be a separate class, and not part of the two-sided pay as clear market.